README.md

```diff
@@ -85,14 +96,14 @@ Qserv install customization is handled with [Kustomize](https://github.com/kuber
 
 ```sh
 # Install a qserv instance with default settings inside default namespace
-kubectl apply -k ./base --namespace='default'
+kubectl apply -k https://github.com/lsst/qserv-operator/overlays/local --namespace='default'
 ```
 
 #### with a Redis cluster
 
 ```sh
 # Install a qserv instance plus a Redis cluster inside default namespace
-kubectl apply -k ./overlays/ci-redis --namespace='default'
+kubectl apply -k https://github.com/lsst/qserv-operator/ci-redis --namespace='default'
```

Comment:
ci-redis is travis?

Reply:
we don't have to change these right now, I think you're trying to converge on naming and settings locations that are more natural along the lines of our conversation at yesterday's coworkers. If the names or explanations can be improved or expanded that would be great, otherwise the kustomizations can be inspected for more detail I think, for now at least.

Reply:
Doc improved here.

Reply:
this bit is not clear `Travis-CI, o, top of kind`